### DISCUSSION TITLE
Save collection in vscode project

### DISCUSSION THREAD

**Discussion by znat** [thunderclient/thunder-client-support#948] [Ideas]
Hi, thanks a lot for this amazing extension!
I'd love to have the possibility to save collections in the current vscode project so they would be versioned and used/edited collaboratively by all team members.

**Comment by rangav** [comment]
Hi @znat Glad you like the extension 

We have this feature already to save to git.
https://github.com/rangav/thunder-client-support#git-sync

Please enable vscode setting `Save to Workspace`

**Comment by smiller0924** [comment]
Where does the Thunder Client 'backup' sync to on GitHub?

**Reply by rangav** [reply]
The extension will save requests data to your current github project, that you opened in vscode

Then you need to check-in to your git server.

**Reply by smiller0924** [reply]
Thank you.  

I now have a Thunder Client in my Linux (Mint/Debian) and Windows PC.  How do I pull down what I have synced to my Git Account to my Windows Thunder Client instance?

**Reply by rangav** [reply]
It’s like your normal git project, from vscode git tab pull changes from windows or Linux
